cli/src/index.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+const execSync = require("child_process").execSync;
+const meow = require("meow");
+const ora = require("ora");
+const download = require("download-git-repo");
+const path = require("path");
+
+/**
+ * A CLI package to help init a bookshop package
+ */
+async function run() {
+  const cli = meow(
+    `
+    Usage
+      $ npx bookshop --init <output-dir>
+ 
+    Examples
+      $ npx bookshop --init .
+  `,
+    {
+      flags: {
+        init: {
+          isRequired: true,
+        },
+      },
+    }
+  );
+
+  const spinner = ora("Downloading bookshop...").start();
+
+  await new Promise((resolve) =>
+    download(
+      "CloudCannon/bookshop",
+      path.resolve(__dirname, "temp"),
+      {},
+      resolve
+    )
+  );
+
+  const output = `${cli.flags.init || "."}`;
+
+  // Copy the example project into user's specified directory
+  try {
+    execSync(`cp -r ${__dirname}/temp/examples/example-bookshop/ ${output}`);
```

Comment:
I had a plan to use [Kickstart](https://github.com/Keats/kickstart) to power this CLI with some interactivity. Currently cloning the example in this repo gives you a lot to then rename or delete.

If you have the gumption to add a template to this repo (separate from the example) that would be fab. I have an example Kickstart project I was playing with for Jekyll that I could show you

Reply:
Cool, I will do this. WIll hit you up if I get stuck

Reply:
Ok, this should be done @bglw .
To test, you just need to link this package locally (npm link @bookshop/cli), and then run
```bash
npx @bookshop/cli --templatePath <path_to_local_example>
```
e.g.
```bash
npx @bookshop/cli --templatePath ../bookshop/examples/example-bookshop-kickstart/
```

Reply:
When merged, just `npx @bookshop/cli` is needed